docs/download.md

```diff
@@ -41,40 +41,18 @@ for your hardware's minimum driver version.
 *For Cloudera and EMR support, please refer to the
 [Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.
 
-### Download v22.12.0
-* Download the [RAPIDS
-  Accelerator for Apache Spark 22.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar)
-
-This package is built against CUDA 11.5 and all CUDA 11.x versions are supported through [CUDA forward
-compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html). It is tested
-on V100, T4, A2, A10, A30 and A100 GPUs with CUDA 11.0-11.5.  For those using other types of GPUs which
-do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or later is required. Users will
-need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
-
-### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 22.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 22.12.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar.asc)
-* Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
-* Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.12.0.jar.asc rapids-4-spark_2.12-22.12.0.jar`
-
-The output if signature verify:
-
-	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
-
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Zstandard compression for Parquet and ORC
-* Support for Hive text reading
-* Improved performance on `like` operations
-* Tiered projections for more expressions to optimize performance  
-* Support for mapInArrow, instr and array_remove operations
-* z-ordering capability on Databricks Delta Lake
-* Dynamic Partition Pruning (DPP) on Databricks 
+* Delta Lake MERGE/DELETE/UPDATE 
+* Function `from_json`
+* Hive text table write 
+* Databricks 11.3 ML LTS support
+* Support batched full join to improve full join's performance
+* Join reordering support  
```

Comment:
Similarly this has not merged.  Curious where you're getting this list from, as ideally it should be based on what's actually in the release, i.e.: from the list of PRs merged into the release line.

Reply:
Join reordering is not in this release, the other functionalities are. 
```suggestion
```

Reply:
removed. @sameerz